go/service/chat_boxer.go

```diff
@@ -0,0 +1,160 @@
+// Copyright 2016 Keybase, Inc. All rights reserved. Use of
```

Comment:
I named it after the type, chatBoxer, which @songgao came up with.  box is the "verb" here: box, unbox.